index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+        rel="stylesheet">
+    <link rel="stylesheet" href="index.css">
+    <title>mobile skeleton</title>
+</head>
+<body>
+<main>
+    <div class="full-wrapper">
+        
+        <nav>
+            <div class="imgs">
+                <img src="/images/Light.png" alt="">
+            </div>
+            <div class="nav-wrapp">
+                <div class="logo">
+                    <span>My Logo</span>
+                </div>
+                <div class="right-nav">
+                    <i id="menu" class="material-icons">menu</i>
+                </div>
+            </div>
+        </nav>
+    
+        <div class="content-wrapper">
+            <div class="header">
+                <span>I’m Desmond
+                    Glad to see you!</span>
+            </div>
+            <div class="about">
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </div>
+            <div class="connect">
+                <span>LET’S CONNECT</span>
+            </div>
+            <div class="socialmedia-icons">
+                <ul>
+                    <li>
+                        <a href="#"><img src="/images/Vector.jpg" alt=""></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="/images/Linkedinicon.jpg" alt=""></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="/images/Vectora.jpg" alt=""></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/Vectorb.jpg" alt=""></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/Vectorc.jpg" alt=""></a>
+                    </li>
+                </ul>
+
+            </div>
+        </div>
+    
+
+     </div>
```

Comment:
- [ ] Please enclose this block in a `section` tag. Currently, you are dealing with the introduction section. In the future, you will add other sections to your portfolio and it is a good idea to group all these with the `section` tag 👍